kbase-extension/static/kbase/js/widgets/appWidgets2/validators/float.js

```diff
@@ -1,21 +1,33 @@
 define(['bluebird'], (Promise) => {
     'use strict';
 
-    function importString(value) {
-        let normalizedValue;
+    function toFloat(value) {
+        const valueType = typeof value;
+        if (valueType === 'number') {
+            return value;
+        } else if (valueType === 'string') {
+            const number = Number(value);
+            if (isNaN(number)) {
+                throw new Error('Invalid float format: ' + value);
+            }
+            return number;
+        }
+        throw new Error('Type ' + valueType + ' cannot be converted to float');
+    }
```

Comment:
Make this act the same way as the `int` validator works -- they were throwing errors in different ways when validation failed.